core/src/com/unciv/logic/map/mapunit/MapUnit.kt

```diff
@@ -854,6 +879,24 @@ class MapUnit : IsPartOfGameInfoSerialization {
         moveThroughTile(tile)
     }
 
+    fun startEscorting() {
+        if (getOtherEscortUnit() != null) {
+            escorting = true
+            getOtherEscortUnit()!!.escorting = true
+        } else {
+            escorting = false
+        }
+        movement.clearPathfindingCache()
+    }
+
+    fun stopEscorting() {
+        if (getOtherEscortUnit() != null) {
+            getOtherEscortUnit()!!.escorting = false
+        }
```

Comment:
Can this be one-lined?
`getOtherEscortUnit()?.escorting = false`
or does that not work?